code/Model/EditableFormField/EditableDateField.php

```diff
@@ -55,7 +55,7 @@ public function getCMSFields()
     public function getFormField()
     {
         $defaultValue = $this->DefaultToToday
-            ? DBDatetime::now()->Format('Y-m-d')
+            ? DBDatetime::now()->Format('yyyy-MM-dd')
```

Comment:
LGTM. Nice catch - confusing that this is in no way the same as PHP `date`.

Reply:
Yep we use CLDR date formats in core now